System.Compression.Primitives/System/IO/Compression/Brotli/BrotliStream.cs

```diff
@@ -10,20 +10,18 @@
 #else
 using nuint = System.UInt32;
 #endif 
-namespace System.IO.Compression.Brotli
+namespace System.IO.Compression
 {
     public partial class BrotliStream : Stream
```

Comment:
But the PR contains non-primitive stream APIs. Also, what do you mean by "using buffers"? what is a "buffer" in your view?